### ISSUE TITLE
Controlling Custom Menu for different users

### ISSUE THREAD

**Issue by decpradeep** [BroadleafCommerce/BroadleafCommerce#1296]
We are using Broadleaf commerce 3.0.2 GA. We are trying to control Left Hand Menu in an admin section for one of the Admin user for the custom controller file. We are expecting to Hide the Particular Menu / Sub Menu for different Users, but we are unable to do it. We are following BLC_ADMIN_SECTION, BLC_ADMIN_MODULE, BLC_ADMIN_SEC_PERM_XREF tables to insert those particular Menus. We got one response before saying " Not to enter the value in BLC_ADMIN_SEC_PERM_XREF table so that it will not come". But we want to have entry in this table but to hide particular menu for particular user,
In other words(Example: 2 users named Mr X & Mr Y and One Menu Named "ABCD". Now our requirement is for Mr X menu ABCD should come but for User Mr Y menu ABCD should not come.)
Please guide us how to do it.

**Comment by bpolster** [comment]
Please direct this type of question to the forums for community assistance.
